src/self-model.scm

```diff
@@ -583,6 +583,7 @@
 (DefineLink
 	(DefinedPredicate "look at person")
 	(SequentialAnd
+		(DefinedPredicate "Time to change gaze")
```

Comment:
We just tested with https://github.com/hansonrobotics/blender_api_msgs/blob/master/cookbook.md#eye-and-head-tracking . As mentioned on the section it is quasi-independent, so this is a valid hack(:smile:), to have gaze work without it being reset by a frequent `look at person` during face-tracking.

So it seems the blender api needs an update, but will check with Wenwai if there are other options.